Makefile

```diff
@@ -89,7 +90,6 @@ errcheck:
 test:
 	go test -race ./...
 
-.tmp/protoc/bin/protoc:
+.tmp/protoc/bin/protoc: ./Makefile ./download_protoc.sh
```

Comment:
I added the deps here, so that the compiler is auto-redownloaded if we change the version (which is in the Makefile) or the download script.

Reply:
👍 nice